src/plugins/intel_npu/src/al/include/intel_npu/weights_pointer_attribute.hpp

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright (C) 2025 Intel Corporation.
+// SPDX-License-Identifier: Apache-2.0
+//
```

Comment:
```suggestion
// Copyright (C) 2018-2025 Intel Corporation
// SPDX-License-Identifier: Apache-2.0
//
```
Check other added files and apply it